.ci/scripts/functional-test.sh

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
```

Comment:
Can we document the input parameters here? This way we could better understand its usage